test/evergreen.yml

```diff
@@ -2618,6 +2618,14 @@ tasks:
               ${test_env_vars|} ${python_binary|python3} ../test/suite/run.py -v 4 test_prepare_hs03.py --hook timestamp
             done
 
+  - name: csuite-long-running
```

Comment:
Does the introduction of this new task mean the `make-check-test` task will not cover those long-running csuite tests? If that's the case we should check all places that call the `make-check-test` task in evergreen.yml and evaluate whether the new task is needed.

Reply:
The `make-check-test` task still works the same and will cover all the csuite tests however the new label `long_running` allows those tests to be excluded from `make-check-test` with the `-LE long_running` ctest argument and then included in the separate task.

Reply:
Ahh, I see. Did not realize "LE" stands up "label-exclude" for ctest.